README.md

```diff
@@ -86,8 +109,4 @@ grunt.initConfig({
 })
 ```
 
-## Contributing
-In lieu of a formal styleguide, take care to maintain the existing coding style. Add unit tests for any new or changed functionality. Lint and test your code using [Grunt](http://gruntjs.com/).
-
-## Release History
-_(Nothing yet)_
+## [Contribute](https://github.com/pdehaan/grunt-standard/blob/master/CONTRIBUTE.md)
```

Comment:
Also, if you use your repo as the authoritative, you'll want to tweak this.
I think you can just shorten to "CONTRIBUTE.md" and it will resolve locally.